interfaces/home.rb

```diff
@@ -0,0 +1,71 @@
+require_relative 'books'
+require_relative 'persons'
+require_relative 'rentals'
+require_relative 'tui'
+
+class App
+  include Tui
+
+  def initialize
+    @options_answer = 0
+    @books = Books.new
+    @persons = Persons.new
+    @rentals = Rentals.new
+    clear_screen
+    main_greeting_message
+    progress_bar
+  end
+
```

Comment:
- [ ] The `App` class should have methods that will return the query results. So when referring to the app inside the `main.rb` file, you will just call the method based on the user choice.

See example:
```ruby

class App
  def initialize
    @books = []
    @people = []
    @rentals = []
  end
  
  # *Good example of Single responsability rule being followed
  def list_books
    @books.each do |book|
      puts book
    end
  end
end
```

See example in main.rb file
```ruby
def main
  app = App.new()
  
   # display details for user to choose from

    case response
        when "1"
          app.list_books()
        when "2"
          # and the flow goes on
   end
```
  

```

Reply:
Hello, @mmsesay, thanks for your useful and kindly feedback!, This problem is solved in commit b28afac.

Reply:
The pleasure is all mine @TheLuisAbarca